apps/saleor-app-checkout/frontend/hooks/useFormattedMessages.ts

```diff
@@ -1,8 +1,8 @@
 import { useRouter } from "next/router";
 import { useMemo } from "react";
 
-import English from "../content/compiled-locales/en.json";
-import Polish from "../content/compiled-locales/pl.json";
+import English from "../../content/compiled-locales/en.json";
+import Polish from "../../content/compiled-locales/pl.json";
```

Comment:
This will cause to download all locales, which could be a problem if we add more of them - maybe we should import them dynamically?

Reply:
Maybe, one day :P

Reply:
Actually that makes sense, I think we had something like that in dashboard

Reply:
https://github.com/saleor/react-storefront/issues/537